device/pensando/arm64-elba-asic-r0/plugins/ssd_util.py

```diff
@@ -0,0 +1,100 @@
+#
+# ssd_generic.py
+#
+# Generic implementation of the SSD health API
+# SSD models supported:
+#  - InnoDisk
+#  - StorFly
+#  - Virtium
+
+try:
```

Comment:
@prgeor , can you check what else plugins are missing from platform side?

Reply:
@ashwin-h please add support for thermalctld, psud, pcied, system eeprom, chassis.py

Reply:
We will enable all the plugins when we enable pmon functionality later.

Reply:
@prgeor has this been answered satisfactorily for you?